hotness/buildsys.py

```diff
@@ -60,7 +62,14 @@ def __init__(self, consumer, config):
 
     def session_maker(self):
         koji_session = koji.ClientSession(self.server, {'timeout': 3600})
-        koji_session.ssl_login(self.cert, self.ca_cert, self.ca_cert)
+        result = koji_session.krb_login(
+            principal=self.krb_principal,
+            keytab=self.krb_keytab,
+            ccache=self.krb_ccache,
+            proxyuser=self.krb_proxyuser
```

Comment:
I do not want any new services that use proxyuser, so I'd suggest to make this non-configurable.
If someone wants to use another user, they can just provide that users' principal and keytab.